src/Mvc/src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/ParameterBinder.cs

```diff
@@ -53,7 +52,7 @@ public class ParameterBinder
         /// <param name="modelMetadataProvider">The <see cref="IModelMetadataProvider"/>.</param>
         /// <param name="modelBinderFactory">The <see cref="IModelBinderFactory"/>.</param>
         /// <param name="validator">The <see cref="IObjectModelValidator"/>.</param>
-        /// <param name="mvcOptions">The <see cref="MvcOptions"/> accessor.</param>
+        /// <param name="mvcOptions">The <see cref="MvcOptions"/> accessor. This parameter is currently unused.</param>
```

Comment:
I'd prefer leaving this doc comment as is. It'd be easy to miss updating this if we do start using it.

Reply:
It's not clear to me this parameter will be used in the future. But, I could move this to `<remarks>`. Either way, the comment is hard to miss.